src/Appointments/Domain/Actions/ValidateClinicDoctorRelation.php

```diff
@@ -11,7 +11,8 @@ class ValidateClinicDoctorRelation
 {
     public function execute(AppointmentDto $appointmentDto): bool
     {
-        return Doctor::whereId($appointmentDto->doctorId)
+        return Doctor::query()
+            ->whereId($appointmentDto->doctorId)
```

Comment:
We try not to use dynamic methods (like methods magically inferred by Laravel by the attributes). Another solution could be this

```suggestion
            ->whereKey($appointmentDto->doctorId)
```